client/components/SingleArtwork.js

```diff
@@ -22,44 +27,70 @@ export class SingleArtwork extends React.Component {
 
     componentDidMount(){
         this.props.getArtwork(this.props.match.params.id)
+        this.getReviews(this.props.match.params.id);
+    }
+
+    async getReviews(artworkId){
+        try {
+            const reviews = (await axios.get(`/api/reviews/${artworkId}`)).data
```

Comment:
Any reason you didn't put this axios call in redux/a reducer? I think it should to be consistent.